settings.gradle

```diff
@@ -2,6 +2,7 @@ include 'common:scala'
 
 include 'core:controller'
 include 'core:dispatcher'
+include 'core:loadBalancer'
```

Comment:
Definitly! @tuler would you mind to copy the code from `loadbalancer` to the controller (they have been merged which is why there was no `build.gradle` left for it). It should then be safe to delete the whole loadbalancer folder I think.